doc/classes/ProjectSettings.xml

```diff
@@ -2364,6 +2364,137 @@
 			[b]Note:[/b] This property is only read when the project starts. To change the physics FPS at runtime, set [member Engine.physics_ticks_per_second] instead.
 			[b]Note:[/b] Only [member physics/common/max_physics_steps_per_frame] physics ticks may be simulated per rendered frame at most. If more physics ticks have to be simulated per rendered frame to keep up with rendering, the project will appear to slow down (even if [code]delta[/code] is used consistently in physics calculations). Therefore, it is recommended to also increase [member physics/common/max_physics_steps_per_frame] if increasing [member physics/common/physics_ticks_per_second] significantly above its default value.
 		</member>
+		<member name="physics/jolt_physics_3d/collisions/active_edge_threshold" type="float" setter="" getter="" default="0.872665">
+			The maximum angle between two triangles (in a [ConcavePolygonShape3D] or [HeightMapShape3D]) within which a convex edge will be considered active or inactive, in radians.
+			Collisions against an inactive edge will have its normal overridden to instead be the surface normal of the triangle. This can help alleviate ghost collisions.
+			[b]Note:[/b] Setting this too high can result in objects not depenetrating properly.
+			[b]Note:[/b] This applies to all shape queries, as well as physics bodies within the simulation.
+			[b]Note:[/b] This does not apply when enabling Jolt's enhanced internal edge removal, which supersedes this.
+			[b]Note:[/b] This setting will only be read once during the lifetime of the application.
+		</member>
+		<member name="physics/jolt_physics_3d/collisions/collision_margin_fraction" type="float" setter="" getter="" default="0.08">
+			The amount of collision margin to use for certain convex collision shapes, such as [BoxShape3D], [CylinderShape3D] and [ConvexPolygonShape3D], as a fraction of the shape's shortest axis, with [member Shape3D.margin] as the upper bound. This is mainly used to speed up collision detection with convex shapes.
+			[b]Note:[/b] Collision margins in Jolt do not add any extra size to the shape. Instead the shape is first shrunk by the margin and then expanded by the same amount, resulting in a shape with rounded corners.
+			[b]Note:[/b] Setting this value too close to [code]0.0[/code] may also negatively affect the accuracy of the collision detection with convex shapes.
+			[b]Note:[/b] This setting will only be read once during the lifetime of the application.
+		</member>
+		<member name="physics/jolt_physics_3d/joints/world_node" type="int" setter="" getter="" default="0">
+			Which of the two nodes bound by a joint should represent the world when one of the two is omitted, as either [member Joint3D.node_a] or [member Joint3D.node_b]. This can be thought of as having the omitted node be a [StaticBody3D] at the joint's position.
+			[b]Note:[/b] In Godot Physics, only [member Joint3D.node_b] can represent the world.
```

Comment:
Add:
Limits are more easily expressed when [member Joint3D.node_a] is the world.